core/server/data/schema/fixtures/fixtures.json

```diff
@@ -412,6 +412,11 @@
                     "name": "Delete labels",
                     "action_type": "destroy",
                     "object_type": "label"
+                },
+                {
+                    "name": "Read member signin url",
```

Comment:
Based on naming convo [above](https://github.com/TryGhost/Ghost/pull/11607/files#r382385753) this could change to `Read member signin`

Reply:
TBH `Read member signin` reads a bit weird to me and looks incomplete, specially when compared to `Read member signin url`.

Reply:
@gargol Just want to confirm that we don't have a corresponding `relations` entry [here](https://github.com/TryGhost/Ghost/blob/master/core/server/data/schema/fixtures/fixtures.json#L591) for Owner because all permissions are by default always implicitly assigned to Owner right?

Reply:
> Just want to confirm that we don't have a corresponding relations entry here

Yes, this is intentional as we only wan to limit access to this resource for Owner users only :+1:

Reply:
Cool thanks 👍